test/headless/specs/utilities.js

```diff
@@ -19,8 +21,9 @@ testModule('elementSatisfiesRatio', class extends TestClass {
         window.STATE.satisfied = true;
       })
     })
-    .wait(30)
+    .wait(constants.time.RAF_THRESHOLD * 5)
```

Comment:
It would be great if we can make it inside `assert`. But the problem what I found by moving it over there is that we need not have to wait for same period in all test cases. In case of impression tests, If we wait little more, test cases fail. And we also need a small time after scroll (it also looks very inconsistent across all place how much exactly we need to wait). So I added all them wait immediately after the scroll. So we wait for small fraction of time then proceed to next action.